test/test_chemistry_rosetta.py

```diff
@@ -0,0 +1,48 @@
+from chemistry import read_PDB, load_rosetta
+from utils import get_fn
+import unittest
+from unittest import skipIf
+
+try:
+    from simtk.openmm.app import PDBFile
+except:
+    PDBFile = None
+    pass
+try:
+    from rosetta import init, pose_from_sequence
+except ImportError:
+    init = pose_from_sequence = None
+    pass
+
+
+def _unpackLen(obj):
+    return len(list(obj))
+
+
+class TestRosetta(unittest.TestCase):
+    """ Tests loading of a Rosetta pose object """
+
+    @skipIf(not init, "Cannot test load_rosetta module without PyRosetta.")
+    def testLoadPose(self):
+        if not init or not pose_from_sequence:
+            raise ImportError('Could not load the PyRosetta module.')
+
+        init()
+        pose = pose_from_sequence(3*'A')
+        struct = load_rosetta(pose)
+        if PDBFile:
+            pdb = PDBFile(get_fn('ala_ala_ala.pdb'))
+            self.assertEqual(_unpackLen(struct.topology.bonds()),
+                             _unpackLen(pdb.topology.bonds()))
+        else:
+            pdb = read_PDB(get_fn('ala_ala_ala.pdb'))
+
+        self.assertEqual(_unpackLen(struct.topology.atoms()),
+                         _unpackLen(pdb.topology.atoms()))
+
```

Comment:
This doesn't actually work if OpenMM is not installed.  You conditionally use `read_PDB` from ParmEd if `PDBFile` from OpenMM is not available, but then you go ahead and request the OpenMM `Topology` instance from the ParmEd PDB file, anyway.  This will raise an `ImportError` if OpenMM is not available.

Also, this only tests against OpenMM's `Topology` class, not really directly against ParmEd's `Structure`.  I'd suggest splitting this in 2 tests, one that tests directly against `Structure.atoms` and `Structure.residues` and the other that tests against OpenMM's `Topology`, and do the `skipIf(not PDBFile)` for the OpenMM test.

The ParmEd test would look something like this:

```Python
    ...
    self.assertEqual(len(struct.atoms), len(pdb.atoms))
    self.assertEqual(len(struct.residues), len(pdb.residues))
    # No test for bonds, yet (it's functionality I hope to add soon)
```
Then the other test would compare `struct` directly with `pdb.topology` from OpenMM.

What do you think?  I'd be happy to submit a PR to you showing you what I mean if you'd like.